src/components/Whiteboard/ElementBehaviors/Selection/types.ts

```diff
@@ -14,8 +14,18 @@
  * limitations under the License.
  */
 
+import { Elements } from '../../../../state/types';
+
 export type WithSelectionProps = {
   readOnly: boolean;
   active: boolean;
   elementId: string;
 };
+
+export type WithExtendedSelectionProps = {
```

Comment:
This should be an intersection type with `WithSelectionProps`.

Reply:
Doh, leveling up my TS skills, thanks!